cowprotocol/cowswap
# feat(hook-store): Add Uniswap v2 withdraw hook dapp

# Summary

Add Uniswap V2 Withdraw Hook Dapp

# To Test

1- Connect your wallet and switch to any uni v2 deployed chain (Arbitrum and Mainnet)
2- In Hooks > Add Pre-Hook Action, the Uni V2 Deposit should appear in the "All hooks" section
3- Fill the form data and add the pre-hook
4- Check with the simulation feature if it is working as expected.

Notes:
- The token list should be the same as the CoW Swap Yield list.
- If the pool that you want to withdraw is not on the token list you can import it by pasting the address on the dropdown menu.
 - This feature should show an error if the pool is not a valid contract or you don't have funds.
- All the UI was already reviewed.
